.github/workflows/reset-default.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  schedule:
+    - cron: '*/1 * * * *'  # Run every 3 minutes
```

Comment:
Are you sure you want to run this job every 3 minutes?
That can be quite awkward for a developer, who inserts data via the gui and develops code to access it.
The script then wipes out the changes every 3 mins.

I would run it once every day, in the middle of the night. For example 30 mins before the automated nightly test.